app/assets/javascripts/components/assignments/assignment.jsx

```diff
@@ -47,6 +48,11 @@ const Assignment = React.createClass({
     if (assignees.length) { assignees.pop(); }
     if (reviewers.length) { reviewers.pop(); }
 
+    let feedback;
+    if (this.props.assignmentGroup.length === 1 || article.article_id) {
```

Comment:
This could use a comment to explain the intention:

If the article exists (and therefore has an article id) then show Feedback, which will be based on the current version of the article. If the article does not exist, then Feedback is based on a user's sandbox, so we only want to show it if there is a single user assigned; otherwise, we don't know which sandbox to base it on.